### PR TITLE
remove data copy from transform and update unit tests

### PR DESCRIPTION
**Pull request by rianfowler**

### CODE REVIEW

File: src/applications/edu-benefits/5490/helpers.jsx [hunk 1]
```diff
@@ -7,14 +7,6 @@ export function transform(formConfig, form) {
   // Clone the form in so we don’t modify the original...because of reasons FP
   const newForm = _.cloneDeep(form);
 
-  // Copy the data if necessary
-  if (form.data['view:currentSameAsPrevious']) {
```
Comment (rianfowler) [inline src/applications/edu-benefits/5490/helpers.jsx]: removed copy behavior from transform

File: src/applications/edu-benefits/tests/5490/schema/expected-data/previous-sponsor-checked.json [hunk 2]
```diff
@@ -15,13 +15,6 @@
     },
     "vaFileNumber": "C12345678"
   },
-  "veteranFullName": {
```
Comment (rianfowler) [inline src/applications/edu-benefits/tests/5490/schema/expected-data/previous-sponsor-checked.json]: expected results in unit tests now only expect `veteranFullName` under `previousBenefits`

**Review by jbalboni — APPROVED** [review]
This looks good to me, though I would like to make sure we test this with the backend and verify that the spool file for an application is the same with and without this change.

**Comment by rianfowler** [comment]
@jbalboni Is there documentation on how to generate the spool file locally or in a test environment?

**Comment by rianfowler** [comment]
@jbalboni Just needs another quick look. I neglected to migrate the `currentSameAsPrevious` from a `view:` 

seems to be working right (some minor differences added so I could make sure I was looking at the right submission)

screen shot 2018-05-16 at 09 34 05

## spool file result on branch 
```
*INIT*
BRANCH

LAST
699999999
499999999
V5490


CH35
*START*
VA Form 22-5490
JAN 2009

         DEPENDENTS APPLICATION FOR EDUCATION BENEFITS
      (Under Provisions of chapters 33 and 35 of Title 38, U.S.C)
      -----------------------------------------------------------

                              BENEFIT ELECTION
                              ----------------

EDUCATION BENEFIT BEING APPLIED FOR: chapter35



                            APPLICANT INFORMATION
                            ---------------------

SSN: 499999999         Sex: M             Date of Birth: 1980-01-01

Name:   branch last

Address:
123 AWESOME
ASDF, AR, 34234
USA

Telephone Numbers:     Primary:
                       Secondary:

Email Address:  test@test.com
Preferred Method of Contact:

Direct Deposit:     Type of Account:
Name of Financial Inst.:
Routing/Transit #:    Account #:

Name, Address, & Telephone Number of Contact:


        Phone:

Relationship to service member:  SPOUSE

Is a divorce or annulment pending?: NO

Have you remarried since the death of the veteran?: N/A

Remarried date:

                         SERVICE MEMBER INFORMATION
                         --------------------------

Name:   sponsorfirst sponsorlast

SSN:   699999999

VA File Number: N/A

Branch of Service:

Veteran's date of birth: 1990-01-01

Date of death or MIA/POW: N/A

Is qualifying individual on active duty: N/A

Outstanding felony or warrant: N/A

                  TYPE AND PROGRAM OF EDUCATION OR TRAINING
                  -----------------------------------------

Type of Education or Training: College

Name and Address of School or Training Establishment:



Date You Started or Will Start Training:

Education or Career Goal:

Restorative training: N/A

Vocational training: N/A

In what state do you anticipate living while participating in this training:

Would you like to receive vocational and educational counseling?: N/A


                    APPLICATION HISTORY
                    -------------------

Previously applied for VA benefits? YES

Previously claimed benefits:
VETERANS EDUCATION ASSISTANCE BASED ON SOMEONE ELSE'S SERVICE: CHAPTER 35 -
SURVIVORS' AND DEPENDENTS' EDUCATIONAL ASSISTANCE PROGRAM (DEA)

Name of individual on whose account previously claimed: sponsorfirst
sponsorlast

SSN of individual on whose account previously claimed:  699999999

VA File Number for previously claimed benefits:


                  ACTIVE DUTY SERVICE INFORMATION
                  -------------------------------

Served on active duty?: NO

Date Entered   Date Separated     Service Component
Active Duty    From Active Duty



                    EDUCATION AND EMPLOYMENT INFORMATION
                    ------------------------------------

Applicant's disposition from high school:

Applicant's date of disposition:

      High School
      ---------------------------
Name and Location of School:


Date of Training:     From:            To:
Hours:
Degree/Diploma/Certificate:

      Education After High School
      ---------------------------


        Employment
        ----------

Since High School
        Principal Occupation:
        Number of Months:
        License or Rating:


          ENTITLEMENT TO AND USAGE OF ADDITIONAL TYPES OF ASSISTANCE
          ----------------------------------------------------------

For Civilian Employees of the U.S. Federal Government Only. Are you receiving
or do you anticipate receiving any money from your agency (including but not
limited to the Government Employees Training Act) for the same period for
which you have applied to the VA for Education Benefits? If you will receive
such benefits during any part of your training, check Yes.    NO

Source of educational assistance for government employee:

      Certification and Signature of Applicant
Signature of Applicant                                               Date

      Certification for Persons on Active Duty
Signature/Title/Branch of Armed Forces Education Service Officer     Date

Electronically Received by VA:  2018-05-16
Confirmation #:  V-EBC-1

*END*
```


## master submission 
```
*INIT*
FIRST

LAST
599999999
499999999
V5490


CH35
*START*
VA Form 22-5490
JAN 2009

         DEPENDENTS APPLICATION FOR EDUCATION BENEFITS
      (Under Provisions of chapters 33 and 35 of Title 38, U.S.C)
      -----------------------------------------------------------

                              BENEFIT ELECTION
                              ----------------

EDUCATION BENEFIT BEING APPLIED FOR: chapter35



                            APPLICANT INFORMATION
                            ---------------------

SSN: 499999999         Sex: M             Date of Birth: 1980-01-01

Name:   first last

Address:
123 AWESOME
ASDF, AR, 34234
USA

Telephone Numbers:     Primary:
                       Secondary:

Email Address:  test@test.com
Preferred Method of Contact: mail

Direct Deposit:     Type of Account:
Name of Financial Inst.:
Routing/Transit #:    Account #:

Name, Address, & Telephone Number of Contact:


        Phone:

Relationship to service member:  SPOUSE

Is a divorce or annulment pending?: NO

Have you remarried since the death of the veteran?: NO

Remarried date:

                         SERVICE MEMBER INFORMATION
                         --------------------------

Name:   sponsorfirst sponsorlast

SSN:   599999999

VA File Number: N/A

Branch of Service:

Veteran's date of birth: 1970-01-01

Date of death or MIA/POW: N/A

Is qualifying individual on active duty: NO

Outstanding felony or warrant: NO

                  TYPE AND PROGRAM OF EDUCATION OR TRAINING
                  -----------------------------------------

Type of Education or Training: College

Name and Address of School or Training Establishment:

        123 AWESOME
        ASDF, AR, 34234
        USA

Date You Started or Will Start Training:

Education or Career Goal:

Restorative training: N/A

Vocational training: N/A

In what state do you anticipate living while participating in this training:

Would you like to receive vocational and educational counseling?: N/A


                    APPLICATION HISTORY
                    -------------------

Previously applied for VA benefits? YES

Previously claimed benefits:
VETERANS EDUCATION ASSISTANCE BASED ON SOMEONE ELSE'S SERVICE: CHAPTER 35 -
SURVIVORS' AND DEPENDENTS' EDUCATIONAL ASSISTANCE PROGRAM (DEA)

Name of individual on whose account previously claimed: sponsorfirst
sponsorlast

SSN of individual on whose account previously claimed:  599999999

VA File Number for previously claimed benefits:


                  ACTIVE DUTY SERVICE INFORMATION
                  -------------------------------

Served on active duty?: NO

Date Entered   Date Separated     Service Component
Active Duty    From Active Duty



                    EDUCATION AND EMPLOYMENT INFORMATION
                    ------------------------------------

Applicant's disposition from high school:

Applicant's date of disposition:

      High School
      ---------------------------
Name and Location of School:


Date of Training:     From:            To:
Hours:
Degree/Diploma/Certificate:

      Education After High School
      ---------------------------


        Employment
        ----------

Since High School
        Principal Occupation:
        Number of Months:
        License or Rating:


          ENTITLEMENT TO AND USAGE OF ADDITIONAL TYPES OF ASSISTANCE
          ----------------------------------------------------------

For Civilian Employees of the U.S. Federal Government Only. Are you receiving
or do you anticipate receiving any money from your agency (including but not
limited to the Government Employees Training Act) for the same period for
which you have applied to the VA for Education Benefits? If you will receive
such benefits during any part of your training, check Yes.    NO

Source of educational assistance for government employee:

      Certification and Signature of Applicant
Signature of Applicant                                               Date

      Certification for Persons on Active Duty
Signature/Title/Branch of Armed Forces Education Service Officer     Date

Electronically Received by VA:  2018-05-16
Confirmation #:  V-EBC-1

*END*
```

**Pull request merged**